Description: Skip MongoDB and DB2 tests
 These both require a running backend, which is not
 possible in Ubuntu main right now. Conditionally
 skip these mixins by only adding them if the associated
 URL environment variable is set.
Author: James Page
Forwarded: no

--- a/ceilometer/tests/db.py
+++ b/ceilometer/tests/db.py
@@ -202,11 +202,6 @@ class MixinTestsWithBackendScenarios(obj
 
     scenarios = [
         ('sqlite', {'db_url': 'sqlite://'}),
-        ('mongodb', {'db_url': os.environ.get('CEILOMETER_TEST_MONGODB_URL')}),
         ('hbase', {'db_url': os.environ.get('CEILOMETER_TEST_HBASE_URL',
                                             'hbase://__test__')}),
-        ('db2', {'db_url': (os.environ.get('CEILOMETER_TEST_DB2_URL') or
-                            os.environ.get('CEILOMETER_TEST_MONGODB_URL',
-                                           '').replace('mongodb://',
-                                                       'db2://'))})
     ]
